src/hdmint/wallet.cpp

```diff
@@ -142,12 +151,30 @@ bool CHDMintWallet::LoadMintPoolFromDB()
 {
     vector<std::pair<uint256, MintPoolEntry>> listMintPool = CWalletDB(strWalletFile).ListMintPool();
 
-    for (auto& mintPoolPair : listMintPool)
+    for (auto& mintPoolPair : listMintPool){
+        LogPrintf("LoadMintPoolFromDB: hashPubcoin: %d hashSeedMaster: %d seedId: %d nCount: %s\n", 
+            mintPoolPair.first.GetHex(), get<0>(mintPoolPair.second).GetHex(), get<1>(mintPoolPair.second).GetHex(), get<2>(mintPoolPair.second));
         mintPool.Add(mintPoolPair);
+    }
 
     return true;
 }
 
+bool CHDMintWallet::GetSerialForPubcoin(const std::vector<std::pair<uint256, GroupElement>>& serialPubcoinPairs, const uint256& hashPubcoin, uint256& hashSerial)
+{
+    bool fFound = false;
+    for(auto serialPubcoinPair : serialPubcoinPairs){
+        GroupElement pubcoin = serialPubcoinPair.second;
```

Comment:
Same here

Reply:
Agreed.

Reply:
Could you clarify what you mean here?

Reply:
pubcoin is a copy now. I think making it ``` GroupElement const & pubcoin = serialPubcoinPair.second``` will be better performance-wise.

Reply:
I think it can be like this  const GroupElement& pubcoin.

Reply:
Will be more readable if we keep this line, only make it const reference.